src/main/java/org/opensrp/web/rest/StockResource.java

```diff
@@ -80,6 +80,14 @@
 
 	private static final String SAMPLE_CSV_FILE = "/importsummaryreport.csv";
 
+	public static final String PAGE_NUMBER = "pageNumber";
+
+	public static final String PAGE_SIZE = "pageSize";
+
+	public static final String ORDER_BY_TYPE = "orderByType";
+
+	public static final String ORDER_BY_FIELD_NAME = "orderByFieldName";
```

Comment:
Define these centrally in `org.opensrp.web.Constants`

Reply:
Done